redirects-list.json

```diff
@@ -24,10 +24,11 @@
   "^/doc/get-started(/.*)?$                                                               /doc/start",
   "^/doc/tutorial(/.*)?$                                                                  /doc/start",
   "^/doc/tutorials/get-started(/.*)?$                                                     /doc/start",
-  "^/doc/tutorials/versioning(/.*)?$                                                      /doc/use-cases/versioning-data-and-model-files/tutorial",
+  "^/doc/tutorials/versioning(/.*)?$                                                      /doc/use-cases/versioning-data-and-models/tutorial",
   "^/doc/tutorials(/.*)?                                                                  /doc/start",
 
-  "^/doc/use-cases/data-and-model-files-versioning/?$                                     /doc/use-cases/versioning-data-and-model-files",
+  "^/doc/use-cases/data-and-model-files-versioning/?$                                     /doc/use-cases/versioning-data-and-models",
+  "^/doc/use-cases/versioning-data-and-model-files(/.*)?$                                 /doc/use-cases/versioning-data-and-models$1",
```

Comment:
Does this change seem correct? I need to double check its behavior on the preview app anyway... ⏳

Reply:
we don't necessarily need to change the URL ... i would not bother

Reply:
OK, reverted this in 579cc5b6 for now.

@jeremydesroches does it matters that the URL path `/doc/use-cases/versioning-data-and-model-files` doesn't exactly match the title `Versioning Data and Models`, SEO-wise? Thanks

Reply:
> does it matter that the URL path `/doc/use-cases/versioning-data-and-model-files` doesn't exactly match the title `Versioning Data and Models`, SEO-wise?

@jorgeorpinel nope, doesn't need to match. The main consideration for URL paths is to keep them short (< 5 words) and coherent (no weird parameters or hashes etc.). Any key terms worth having in the title should be carried into the URL if possible, as you have done. This one checks out. 👍